app/views/merchant/items/new.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1>Create New Item For <%=link_to @merchant.name, "/merchants/#{@merchant.id}" %></h1>
 
+
 <%= render partial: 'form', locals: { path: merchant_items_path,
                                       method: :post,
                                       button_name: "Add Item",
-                                      item: params } %>
+                                      item: @item ||= @merchant.items.new() } %>
```

Comment:
I thought form_for would've taken care of this - why would you have to still pass it an empty object?